### PR TITLE
minor redundant code fix

### PR DESCRIPTION
**Pull request by axif0**
### Contributor checklist

- [x] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch

---

### Description
We can reuse [cli_utils](https://github.com/scribe-org/Scribe-Data/blob/9e9ceac3158deea011752c02f9fef4e01705b7c1/src/scribe_data/cli/cli_utils.py#L29) code in [list.py](https://github.com/scribe-org/Scribe-Data/blob/9e9ceac3158deea011752c02f9fef4e01705b7c1/src/scribe_data/cli/list.py#L29)

### CODE REVIEW

File: src/scribe_data/load/data_to_sqlite.py [hunk 1]
```diff
@@ -324,7 +324,6 @@ def table_insert(data_type, keys):
             )
 
     print("Database creation/update process completed.")
-    print("Database creation/update process completed.")
```
Comment (andrewtavis) [inline src/scribe_data/load/data_to_sqlite.py:327]: Nice catch!

**Comment by andrewtavis** [comment]
Can you check out the tests here, @axif0 :) I'm not sure that we can import variables from another files that are defined with `with ... as ...`. Maybe you could remove those and just have this be to remove the repeat print statement? Happy to hear your thoughts though!

**Comment by axif0** [comment]
@andrewtavis  My bad I forgot to check the tests. Can you please check it?

**Review by andrewtavis — APPROVED** [review]
All seems to be working well now, @axif0! Appreciate all the cleanup you're doing! 🚚😊

**Pull request merged**